src/ngraph/runtime/aligned_buffer.cpp

```diff
@@ -35,6 +36,10 @@ runtime::AlignedBuffer::AlignedBuffer(size_t byte_size, size_t alignment)
     {
         size_t allocation_size = m_byte_size + alignment;
         m_allocated_buffer = static_cast<char*>(malloc(allocation_size));
+        if (m_allocated_buffer == nullptr)
```

Comment:
Throw is very likely to fail if `malloc` has failed, but sometimes you get lucky.

Reply:
Is "NGRAPH_ERR + exit(1)" better than throw?

Reply:
I would throw an std::bad_alloc, or if you really want to go to the effort you can throw an ngraph_bad_alloc which does not yet exist.
The code will exit one way or the other and as @diyessi says you might get lucky. Throw is the correct behavior.